File: fixtures/scale/objc/large/Families/Gronsfeld/GronsfeldVectors.m
#import "GronsfeldVectors.h"
#import "CLTestVector.h"

@implementation GronsfeldVectors

+ (NSArray *)all {
    NSMutableArray *vectors = [NSMutableArray array];
        [vectors addObject:[CLTestVector vectorWithPlaintext:@"ATTACKATDAWN" expected:@"DYAJNXPKWVTM"]];
        [vectors addObject:[CLTestVector vectorWithPlaintext:@"THEQUICKBROWNFOX" expected:@"WMLZFVRBUMLVOITE"]];
        [vectors addObject:[CLTestVector vectorWithPlaintext:@"DEFENDTHEEASTWALL" expected:@"GJMNYQIYXZXRUZFSU"]];
    return vectors;
}

@end

File: fixtures/scale/swift/large/Families/Gronsfeld/GronsfeldVectors.swift
// Known-answer test data captured from GronsfeldCipher's own encode().
enum GronsfeldVectors {
    static func all() -> [TestVector] {
        return [
            TestVector(plaintext: "ATTACKATDAWN", expected: "DYAJNXPKWVTM"),
            TestVector(plaintext: "THEQUICKBROWNFOX", expected: "WMLZFVRBUMLVOITE"),
            TestVector(plaintext: "DEFENDTHEEASTWALL", expected: "GJMNYQIYXZXRUZFSU"),
        ]
    }
}
